pkgs/top-level/python-packages.nix

```
@@ -5426,8 +5426,12 @@ self: super: with self; {
 
   ledger_agent = callPackage ../development/python-modules/ledger_agent { };
 
+  ledger_bitcoin = callPackage ../development/python-modules/ledger_bitcoin { };
```

use dash instead of underscore

```suggestion
  ledger-bitcoin = callPackage ../development/python-modules/ledger-bitcoin { };
```